client/my-sites/plans-features-main/hooks/use-experiment-for-trail-map.ts

```diff
@@ -1,14 +1,48 @@
 import config from '@automattic/calypso-config';
+import { useExperiment } from 'calypso/lib/explat';
 import type { DataResponse } from '@automattic/plans-grid-next';
 
 interface Props {
 	flowName?: string | null;
 }
 
-function useExperimentForTrailMap( { flowName }: Props ): DataResponse< boolean > {
+export enum TrailMapVariant {
+	Control = 'control',
+	TreatmentCopy = 'treatment-copy',
+	TreatmentStructure = 'treatment-structure',
+	TreatmentCopyAndStructure = 'treatment-copy-and-structure',
+}
```

Comment:
Minor nit, it could be simpler to use a simple string type here. Eg:

```ts
export type TrailMapVariant = 'control' | 'treatment-copy' | 'treatment-structure' | 'treatment-copy-and-structure';
```